lib/ddtrace/contrib/rails/patcher.rb

```diff
@@ -11,10 +11,9 @@ module Patcher
         option :auto_instrument_redis, default: false
```

Comment:
@p-lambert do we need these `auto_instrument_*` fields? I think we can get rid of that since the configuration now is explicit and not at import-time.

Reply:
https://github.com/DataDog/dd-trace-rb/pull/257